cmd/generator/main.go

```diff
@@ -23,5 +24,11 @@ func main() {
 	if err != nil {
 		panic(fmt.Sprintf("cannot calculate the absolute path with %s", rootDir))
 	}
-	pipeline.Run(config.GetProvider(), absRootDir)
+	p := config.GetProvider()
+	pipeline.Run(p, absRootDir)
+	if fp := os.Getenv("SKIPPED_RESOURCES_CSV"); len(fp) != 0 {
```

Comment:
You may consider using a cli parsing library and accept both flags and env vars to keep argument parsing consistent like in [uptest](https://github.com/upbound/uptest/blob/581f891e0a73d6d9dd236a66e07f72c5047d9ad3/cmd/main.go#L23).

But it is totally up to you to deal with it now or later if we need more inputs. But at least a comment on why we use env here would be helpful.

Reply:
Done. Moved to the `kingpin.v2` package for argument parsing in the generator.